### PR TITLE
[1807] Add accrediting_providers to provider enrichment form

### PR DESCRIPTION
**Pull request by dankmitchell**
~❗️  Cannot be merged until https://github.com/DFE-Digital/manage-courses-backend/pull/680 is done ❗️~

### Guidance to review
# After
localhost_3000_organisations_1JA_2019_details(Laptop with MDPI screen)

localhost_3000_organisations_1JA_2019_about(Laptop with MDPI screen) (1)

### CODE REVIEW

**Comment by dankmitchell** [comment]
[About your accredited body (pull onto Rails)](https://trello.com/c/ATAPQ9c6/1807-about-your-accredited-body-pull-onto-rails)

File: app/views/providers/details.html.erb [hunk 1]
```diff
@@ -36,6 +36,9 @@
       <dl class="govuk-summary-list">
         <%= enrichment_summary_item('About your organisation', @provider.train_with_us, 'train_with_us') %>
         <%= enrichment_summary_item('Training with disabilities and other needs', @provider.train_with_disability, 'train_with_disability') %>
+        <% @provider.accrediting_providers.each do |provider| %>
```
Comment (defong) [inline app/views/providers/details.html.erb]:
```suggestion
        <% @provider.accrediting_providers.each do |accrediting_provider| %>
```

`get_provider_name(provider['UcasProviderCode']` still think backend should sort it all out for you

**Comment by NyWeb** [comment]
[[S] Travis randomly fails on mcb courses list spec](https://trello.com/c/Q2kyykcw/1843-s-travis-randomly-fails-on-mcb-courses-list-spec)

File: app/controllers/providers_controller.rb [hunk 2]
```diff
@@ -39,6 +39,7 @@ def build_provider
     )
 
     @provider = Provider
+      .includes(:accrediting_providers)
```
Comment (timabell) [inline app/controllers/providers_controller.rb]: :tada:

File: app/helpers/provider_helper.rb [hunk 3]
```diff
@@ -17,4 +17,12 @@ def google_form_url_for(settings, email, provider)
...
       { settings.email_entry => email, settings.provider_code_entry => provider.provider_code }.to_query
   end
+
+  def get_provider_name(provider_code, cycle_year)
+    Provider
+      .where(year: cycle_year)
+      .find(provider_code)
+      .first
+      .provider_name
+  end
```
Comment (timabell) [inline app/helpers/provider_helper.rb]: Can this be deleted now?

Comment (dankmitchell) [reply app/helpers/provider_helper.rb]: Sorted

**Comment by defong** [comment]
have we already have test for this?

**Comment by tvararu** [comment]
[[S] Update accredited bodies when updating provider enirchment](https://trello.com/c/b2gcKPd7/1901-s-update-accredited-bodies-when-updating-provider-enirchment)

**Comment by timabell** [comment]
I declare this awesome

**Pull request merged**